Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "themis",
+    products: [
+        .library(
+            name: "themis",
+            targets: ["themis"]),
+    ],
+    dependencies: [],
```

Comment:
At first I wanted to ask, “Where is `cl-openssl`?“ But then I realized that it's not here because themis.xcframework will be statically linked to openssl.xcframework, hence users of themis.xcframework don't need to install shared OpenSSL.